src/packages/cli/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.0.0-alpha.1284",
+  "version": "2.0.0-dev.0",
```

Comment:
Is this correct? I was expecting `2.1.0-dev.0` here.

Reply:
The value in here doesn't matter, as it will be overridden with `2.1.0-dev.1`